src/main/java/uk/gov/companieshouse/confirmationstatementapi/controller/EmailController.java

```diff
@@ -18,10 +16,10 @@
 import java.util.HashMap;
 
 import static uk.gov.companieshouse.confirmationstatementapi.utils.Constants.ERIC_REQUEST_ID_KEY;
-import static uk.gov.companieshouse.confirmationstatementapi.utils.Constants.TRANSACTION_ID_KEY;
+import static uk.gov.companieshouse.confirmationstatementapi.utils.Constants.COMPANY_NUMBER;
 
 @RestController
-@RequestMapping("/transactions/{transaction_id}/confirmation-statement")
```

Comment:
Changing the URL here has an impact on the Traefik rules required in the docker-compose for this module (= task ticket required on CHS Dev Env project).  During my testing with Postman I was initially getting 404 not found !!! urgh the valid return meaning blank / empty email BUT perhaps was the URL incorrect?!?,  I guessed the latter and added this rule to docker-compose and restarted then I was able to retrieve the email in the KERMIT2 database :-(
 || PathPrefix(`/private/confirmation-statement`)

Reply:
Yes - I have added the path to the routes.yaml and have a note to update docker-compose once this is merged, but that will be a separate PR as it's in a different repo.

Reply:
OK, (may be "have a note to update docker-compose once this is merged" could also be added to comments of ticket 716 and/or sub-task created)